geonode/static/geonode/js/messages/message_recipients_autocomplete.js

```diff
@@ -11,21 +11,22 @@ function get_groups_data(data) {
 }
 
 class MessageRecipientsTags {
-    constructor(input, data_extract_func, url, blacklist=[]) {
+    constructor(input, data_extract_func, url, blacklist = []) {
         if (input.tagName !== 'INPUT' || input.type !== 'text') {
             throw Error('Base element should be <input type="text">');
         }
         this.input = input;
         this.tagify = null;
-        this.controller = null;
         this.data_extract_func = data_extract_func;
         this.url = url;
         this.blacklist = blacklist;
+        this.request_controller = null;
     }
 
     init() {
         this.tagify = new Tagify(this.input, {whitelist: [], blacklist: this.blacklist});
         this.tagify.on('input', this._onInputHandler.bind(this));
+        this.request_controller = new AbortController();
```

Comment:
AbortController isn't supported on IE. I don't want to introduce breaks in support for such small issues, and in any case a compatibilty break must be discussed with the community.

Reply:
we've decided to drop support for IR with GEoNode 3, so I'm fine with this :)